Gemfile

```diff
@@ -25,18 +27,26 @@ gem 'rubocop-rails', require: false
 gem 'bootsnap', '>= 1.4.4', require: false
 
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
-# gem 'rack-cors'
+gem 'rack-cors'
 
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem 'rspec-rails', '~> 5.0.0'
+  gem 'pry-rails'
 end
 
 group :development do
   gem 'annotate'
   gem 'brakeman'
 end
 
+group :test do
+  gem 'shoulda-matchers', '~> 5.0'
+  gem 'factory_bot_rails'
+  gem 'faker', :git => 'https://github.com/faker-ruby/faker.git', :branch => 'master'
```

Comment:
Is ` :git => 'https://github.com/faker-ruby/faker.git', :branch => 'master'` needed ?